packages/coil-extension/src/util/calculateMaxAllowableTip.ts

```diff
@@ -0,0 +1,29 @@
+export const calculateMaxAllowableTip = (
```

Comment:
If not already done, this needs unit tests.

Reply:
already done 👍